src/containers/BookList.js

```diff
@@ -1,24 +1,27 @@
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import Book from '../components/Book';
-import { createBookAction, removeBookAction } from '../actions';
 
 const BookList = ({ books }) => {
   const bookRows = books.map(book => (<Book key={`book-number-${book.bookId}`} book={book} />));
```

Comment:
Here you are passing all books to the Book component, instead of passing the filtered results based on the category selected. ie
`const visible = books.filter(book => book.category === filter || filter === 'All')`
 then 
`visible.map(book => (<Book key={`book-number-${book.bookId}`} book={book} />));`

Reply:
@Forison, thank you for this advice but I think the above comment should be implemented in the next milestone, this mileston 2, I think should be for Book form and implementing the remove book, not yet filter. The filter should be for the next milestone.

Reply:
@descholar-ceo, I see milestone 3 on my dashboard, are you sure you selected the right milestone whilst, submitting? 
check below
![m33](https://user-images.githubusercontent.com/29695661/102726913-4a3c9280-42f0-11eb-8c99-b1a2c2dbcb82.png)
Kindly make sure to select the right milestone when submitting, else the reviewer may review based on the milestone he has. Let me know if perhaps it is a bug from our side so, I can draw the attention of the relevant authorities. I will go ahead and re-review based on milestone 2.🤗

Reply:
It's milestone 3,I'm sorry it's me who confused them in the previous comment. So, actually the things of filtering are on milestone 4, so, I didn't implement the milestone requirements in milestone 3. My apologies for the confusion. Check the milestone 3 requirements!

Reply:
@Forison, or if it's me who's mistaken with understanding the requirements, I'm open and flexible to re-implement this! Thanks in advance!